ChiModules/LinearBoltzmannSolver/lbs_01d_init_spat_discr.cc

```diff
@@ -20,10 +19,112 @@ void lbs::SteadySolver::InitializeSpatialDiscretization()
     chi_math::SpatialDiscretization_PWLD::New(grid, COMPUTE_CELL_MAPPINGS |
                                                     COMPUTE_UNIT_INTEGRALS);
 
+  ComputeUnitIntegrals();
+}
+
+void lbs::SteadySolver::ComputeUnitIntegrals()
+{
+  chi::log.Log() << "Computing unit integrals.\n";
+  const auto& sdm = *discretization;
+
+  const size_t num_local_cells = grid->local_cells.size();
```

Comment:
for my education, why grid-> and not grid. ? I suppose grid is really a grid_ptr, right?

Reply:
Yes. Like I mentioned above. I will fix this notation to be more consistent.